lib/keycloak.d.ts

```diff
@@ -348,6 +348,11 @@ export interface KeycloakRoles {
 	roles: string[];
 }
 
+export interface KeycloakUserInfo {
+	sub?: string;
```

Comment:
From what I am gathering from [the specification](https://openid.net/specs/openid-connect-core-1_0.html#rfc.section.5.3.2), in all circumstances the `sub` field should always be present, why is it optional here?

> The `sub` (subject) Claim MUST always be returned in the UserInfo Response.

```suggestion
	sub: string;
```

Reply:
I see what you're saying, specification of AccessToken does not says that `sub` must be there.

In addition, `KeycloakUserInfo` should have `iss` (issuer) and `aud` (audience) as specified here:

> If signed, the UserInfo Response MUST contain the Claims iss (issuer) and aud (audience) as members. The iss value MUST be the OP's Issuer Identifier URL. The aud value MUST be or include the RP's Client ID value.

Reply:
Looks like we don't have support for signed responses here at the moment, so I believe we can omit the `iss` and `aud` for now. Additionally, it looks like we're missing some elements required by the spec for validating the `sub` as well:

> NOTE: Due to the possibility of token substitution attacks (see [Section 16.11](https://openid.net/specs/openid-connect-core-1_0.html#TokenSubstitution)), the UserInfo Response is not guaranteed to be about the End-User identified by the `sub` (subject) element of the ID Token. The `sub` Claim in the UserInfo Response MUST be verified to exactly match the `sub` Claim in the ID Token; if they do not match, the UserInfo Response values MUST NOT be used.

However, I would propose we handle this in a subsequent PR, if we do decide to handle this at all. So for now this is good enough in my book.